dataset: sick	No info from HuggingFace
dataset: sick	splits: train: 4439 rows, test: 4906 rows, val: 495 rows
dataset: sick	split: train	columns: id, sentence_A, sentence_B, label, relatedness_score, entailment_AB, entailment_BA, sentence_A_original, sentence_B_original, sentence_A_dataset, sentence_B_dataset
dataset: sick	labels: entailment, neutral, contradiction
***ERROR***
model: None
data: H:sick
eqn: None
splits: train
split: train
task: None
error: (<class 'requests.exceptions.HTTPError'>, HTTPError('405 Client Error: Not Allowed for url: https://huggingface.co/api/models-tags-by-type'), <traceback object at 0x7f24a25c0040>)
